ibis/backends/tests/test_numeric.py

```diff
@@ -527,20 +527,14 @@ def test_sa_default_numeric_precision_and_scale(
 
 @pytest.mark.notimpl(["dask", "datafusion", "impala", "pandas", "sqlite", "polars"])
 @pytest.mark.notyet(
-    ["clickhouse", "snowflake"],
-    reason="backend doesn't implement a [0.0, 1.0) random function",
+    ["clickhouse"],
+    reason="backend doesn't implement a [0.0, 1.0) or [0.0, 1.0] RANDOM() function",
```

Comment:
Should we also update docstring of `random` function?
https://github.com/ibis-project/ibis/blob/409f49709d90ffb1af480cfa4dd8ea62597cba32/ibis/expr/api.py#L602

Reply:
Good idea! We can do that in a follow-up.